### DISCUSSION TITLE
Load files in project directory

### DISCUSSION THREAD

**Discussion by samsze1** [tauri-apps/tauri#7199] [Q&A]
I am trying to read a file (`fs.readTextFile`) located relative to the project directory. I am aware of `$RESOURCE` but `$RESOURCE` is only useful when I am testing a build. Is there a way to add the current project file to `fs > scope` during dev? Thank you 🙏 

Sidenote: I also tried working around it by creating a symlink from `$DOCUMENT` to where the file is located, but I am still getting the `Unhandled Promise Rejection: path not allowed on the configured scope` error 😅 

<img width="677" alt="image" src="https://github.com/tauri-apps/tauri/assets/115963458/f499c227-967d-41ef-b701-4811c3f10cb4">

**Comment by FabianLars** [comment]
I think that will only work in Rust where the cwd (current working dir) is set correctly. I don't think we plan to add support for this either since it's generally fairly limited in use.

**Reply by samsze0** [reply]
Make sense. Thank you for the reply
